src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/repository/DraftReferralRepository.kt

```diff
@@ -0,0 +1,22 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.repository
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.DraftReferral
+import java.util.Optional
+import java.util.UUID
+
+interface DraftReferralRepository : JpaRepository<DraftReferral, UUID> {
+
+  // queries for draft referrals
+  override fun findById(id: UUID): Optional<DraftReferral>
```

Comment:
Do we have to define this, if this is `override`?

Reply:
we don't need the override, removed it